app/grub_control.py

```diff
@@ -1,15 +1,17 @@
 import re
+import os
 import subprocess
 import shlex
 import tempfile
 import shutil
-import os
 from pathlib import Path
 from pprint import pformat
 
-from ota_error import OtaErrorUnrecoverable
-import configs as cfg
 import log_util
+import configs
+from ota_error import OtaErrorUnrecoverable
+
+cfg = configs.create_config("grub")
```

Comment:
Ah the reason for this is that grub_control must use GrubControlConfig, so we explicitly create a GrubControlConfig instance for grub_control module to avoid potential problems caused by initializing grub_control module with mismatching config.